backend/app/tests/crud/test_inquiry.py

```diff
@@ -13,3 +14,14 @@ def test_create_inquiry(db: Session) -> None:
     inquiry_in = InquiryCreate(text=text)
     inquiry = crud.create_inquiry(session=db, inquiry_in=inquiry_in)
     assert inquiry.text == text
+
+
+def test_get_inquiry(db: Session) -> None:
+    length = random.randint(10, 255)
+    text = "".join(random.choices(string.ascii_lowercase, k=length)).capitalize()
+    inquiry_in = InquiryCreate(text=text)
+    created_inquiry = crud.create_inquiry(session=db, inquiry_in=inquiry_in)
```

Comment:
This will create an entry in the actual database. I am not sure we should do this. I did the same thing in my code but I think I need to rewrite my test.   Probably you can mock the call to create_inquiry

https://pytest-with-eric.com/mocking/pytest-mocking/

I think you would want something like this (but I did not look at any docs so don't copy this exactly)
```
    mocker.patch(
    "app.models.crud.Inquiry.create_inquiry"
    , {"id": "6",
       "text": "Why do birds suddenly appear every time you are near?",
       "date": "2024-09-17"
       })
```

Reply:
Yes, I think we need to come up with better unit testing strategy. During the time I was implementing this, `pytest-mock` wasn't available in our project. So, once included, I plan to revise my unit tests.